### DISCUSSION TITLE
Open Source Forum 2022

### DISCUSSION THREAD

**Discussion by scott-wilson** [vfx-rs/organization#15] [General]
The 2022 ASWF open source forum is happening on March 10th, 2022 (https://events.linuxfoundation.org/aswf-open-source-forum/).

We've been asked to create a 5 minute presentation. The rough format is...

- What we achieved?
- What is in the plan?

My initial thoughts are something along this:

- What we achieved?
  - Created an alpha binding of OpenEXR
  - Brought a lot of excitement about Rust in the VFX industry.
- What is in the plan?
  - Have the OpenEXR binding an official part of the OpenEXR project.
  - Have more ASWF projects bound by SIGGRAPH
  - Assist other projects interested in Rust with converting code to Rust and/or creating bindings

What's everyone's thoughts?

**Comment by davvid** [comment]
I'm committed to contributing this line item for the "What we achieved?" category in time for the summit:

* Created an alpha ptex-rs high-level binding for Ptex

**Comment by anderslanglands** [comment]
I’d also consider oiio in alpha state. Openexr I’d call beta.
